similaritymeasures/similaritymeasures.py

```diff
@@ -879,3 +879,125 @@ def dtw_path(d):
     path = np.array(path)
     # reverse the order of path, such that it starts with [0, 0]
     return path[::-1]
+
+def mae(exp_data, num_data, metric='euclidean', **kwargs):
```

Comment:
So MAE and MSE are special cases of computing the distance for particular metrics. MAE is the mean of the 'cityblock' (`p=1`) metric, and MSE is the mean of the 'euclidean' (`p=2`) metric. Thus it doesn't make sense to me that you would want to expose the `metric` and `kwargs` here. I'll explain what I think you should do in another comment.